templates/pipelines/pipeline_builder.html

```diff
@@ -21,6 +21,14 @@
   </style>
 {% endblock page_head %}
 {% block breadcrumbs %}
+  {% if origin == "chatbots" %}
+    <div class="text-sm breadcrumbs" aria-label="breadcrumbs">
+      <ul>
+        <li><a href="{% url 'chatbots:chatbots_home' request.team.slug %}">Chatbots</a></li>
+        <li class="pg-breadcrumb-active" aria-current="page">{{ experiment.name }}</li>
```

Comment:
This should have another part for the chatbot details page:

```suggestion
        <li><a href="{% url 'chatbots:single_chatbot_home' request.team.slug experiment.id %}">{{ experiment.name }}</a></li>
        <li class="pg-breadcrumb-active" aria-current="page">Edit</li>
```

Reply:
done